packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -66,68 +67,85 @@ export default class HostItem extends Component {
+      power_state: powerState,
+      tags,
+      version,
+    } = host
+    const { size, usage } = memory
     const toolTipContent =
-      host.power_state === `Running` && !host.enabled
+      powerState === `Running` && !enabled
         ? `disabled`
-        : _(`powerState${host.power_state}`)
+        : _(`powerState${powerState}`)
     return (
       <div className={styles.item}>
-        <BlockLink to={`/hosts/${host.id}`}>
+        <BlockLink to={`/hosts/${id}`}>
           <SingleLineRow>
             <Col smallSize={10} mediumSize={9} largeSize={3}>
               <EllipsisContainer>
                 <input
                   type='checkbox'
                   checked={selected}
                   onChange={this._onSelect}
-                  value={host.id}
+                  value={id}
                 />
                 &nbsp;&nbsp;
                 <Tooltip
                   content={
-                    isEmpty(host.current_operations) ? (
+                    isEmpty(currentOperations) ? (
                       toolTipContent
                     ) : (
                       <div>
                         {toolTipContent}
                         {' ('}
-                        {map(host.current_operations)[0]}
+                        {map(currentOperations)[0]}
                         {')'}
                       </div>
                     )
                   }
                 >
                   {!isEmpty(host.current_operations) ? (
                     <Icon icon='busy' />
-                  ) : host.power_state === 'Running' && !host.enabled ? (
+                  ) : powerState === 'Running' && !enabled ? (
                     <Icon icon='disabled' />
                   ) : (
-                    <Icon icon={`${host.power_state.toLowerCase()}`} />
+                    <Icon icon={`${powerState.toLowerCase()}`} />
                   )}
                 </Tooltip>
                 &nbsp;&nbsp;
                 <Ellipsis>
                   <Text
-                    value={host.name_label}
+                    value={nameLabel}
                     onChange={this._setNameLabel}
                     useLongClick
                   />
                 </Ellipsis>
                 &nbsp;
-                {container && host.id === container.master && (
+                {container && id === container.master && (
                   <span className='tag tag-pill tag-info'>
                     {_('pillMaster')}
                   </span>
                 )}
                 &nbsp;
                 {this.props.needsRestart && (
                   <Tooltip content={_('rebootUpdateHostLabel')}>
-                    <Link to={`/hosts/${host.id}/patches`}>
+                    <Link to={`/hosts/${id}/patches`}>
                       <Icon icon='alarm' />
                     </Link>
                   </Tooltip>
                 )}
                 &nbsp;
+                <AssertConsistentHostServerTime host={id} />
```

Comment:
Simply `host.id` to avoid unrelated changes.

Reply:
I do it  to not access to `id` multiple.

Reply:
It not cost a lot of performance and it produce a lot of unrelated change. Better use `host.id` to avoid it.